src/main/java/de/embl/cba/mobie/n5/zarr/N5OmeZarrReader.java

```diff
@@ -69,6 +71,9 @@ static private GsonBuilder initGsonBuilder(final GsonBuilder gsonBuilder) {
         gsonBuilder.registerTypeAdapter(DType.class, new DType.JsonAdapter());
         gsonBuilder.registerTypeAdapter(ZarrCompressor.class, ZarrCompressor.jsonAdapter);
         gsonBuilder.serializeNulls();
+        gsonBuilder.registerTypeAdapter(ZarrAxes.class, new ZarrAxesAdapter());
+        gsonBuilder.registerTypeAdapter(N5Reader.Version.class, new VersionAdapter());
+        gsonBuilder.setPrettyPrinting();
```

Comment:
@KateMoreva This is because this gson is used for both reading and writing jsons. When writing a json, the pretty printing means it will be nicely indented/formatted to make it more readable. Otherwise, it writes everything to one long line which is a bit difficult to read.

Reply:
Ok, thank you. As N5OmeZarrReader is similar to N5S3ZarrReader, I do not think that it is a problem if N5S3ZarrReader will also have prettyPrinting.

Reply:
Yes - there shouldn't be a problem with that :)